### PR TITLE
[darwin] Inspector tools should also be included into a debug build

### PR DESCRIPTION
**Pull request by stffabi**

### CODE REVIEW

**Comment by stffabi** [comment]
Just found this after testing the refactor cli branch 😄

File: v2/internal/frontend/desktop/darwin/inspector_dev.go
Comment (leaanthony) [inline v2/internal/frontend/desktop/darwin/inspector_dev.go:5]: Should we output this as a warning when compiling with debug?

**Review by leaanthony — APPROVED** [review]
Just a small comment.

File: v2/internal/frontend/desktop/darwin/inspector_dev.go
Comment (stffabi) [reply v2/internal/frontend/desktop/darwin/inspector_dev.go:5]: Yeah we can do that. I'll wait until the cli refactor has been merged so we don't get conflicts.

Comment (leaanthony) [reply v2/internal/frontend/desktop/darwin/inspector_dev.go:5]: Merged

Comment (stffabi) [reply v2/internal/frontend/desktop/darwin/inspector_dev.go:5]: Added the warning, what do you think?

File: v2/pkg/commands/build/base.go
Comment (leaanthony) [inline v2/pkg/commands/build/base.go:564]: 🙏

**Review by leaanthony — APPROVED** [review]

**Comment by stffabi** [comment]
Thanks for approving 🙏

**Pull request merged**
